leading-teams/peer-level-meeting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Peer-level meeting
+meta_description: >
+  Nebulab team leads have a weekly discussion in which they share problems and
+  solutions. Read our Playbook to learn how we keep a shared team culture and
+  support each other!
+---
+
+Each client has its own story and unique challenges, in order to share problems,
+find solutions together, and  keep teams from being siloed, team leads will
+check-in once a week in a peer-level meeting.
```

Comment:
"check in", without a dash. "Check-in" is a noun, "check in" is a verb 🙂 

Also, I've always used "to check in" in the sense of monitoring someone/something's progress. The goal of this meeting is more to have a chat and discuss problems.